extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -46,6 +47,7 @@ export class ServerPropertiesDialog extends ObjectManagementDialogBase<Server, S
 	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
 		super(objectManagementService, options);
 		this.dialogObject.customButtons[1].enabled = false;
+		this.dialogObject.okButton.label = uiLoc.ApplyText;
```

Comment:
I'm not sure if changing the ok button label, should be done only properties. Maybe we can add it to the other dialogs as well?

Reply:
For now I have added it to the Server/Database props and as 'Create' for create database. And yes, it would be nice to have this across the ADS.

@Charles-Gagnon / @corivera / @alanrenmsft, The validation/saving and then closing dialog is applied for current admin workstream, do you guys think we can apply this to all dialogs? users/logins/etc etc

Reply:
+1, we should have consistent experiences and behaviors

I would actually suggest removing this change until we agree on how broadly to apply this - and then go in and update everything at once. This way we aren't left with one dialog being different than the others while we discuss.

Reply:
Sure, I'm not going to merge this change, but will add a commit that makes this logic consistent to all objectManagement dialogs. People can review and share their thoughts and then we can go forward with this PR. Thanks for your inputs here :)

Reply:
Yeah, I agree with that

Reply:
Can you do that in a separate branch? We should do that update in a separate PR - so maybe just focus on the other logic in this PR and we can get that in first.

Reply:
Removed the button label changes from the PR.

These limited dialogs still use the onConfirmation() override method to validate and save changes while dialog is in open state. All other dialogs go as is from the initialize() method.